src/server/models/users.model.js

```diff
@@ -2,13 +2,12 @@
 
 import mongoose from 'mongoose';
 
-/**
+/*
  * Define the schema model for user.
  */
 
 const Schema = mongoose.Schema;
 let userSchema = new Schema({
-  userId: String,
```

Comment:
Yes, we can use that: ensure we copy _id into our custom field user_id though. Otherwise it's a weird hashCode for the object which will end up being difficult to work with in some other context.